app/assets/stylesheets/components/_footer.scss

```diff
@@ -76,44 +78,127 @@ footer {
       }
   
       .social {
+        border-bottom: $simple-boder-black;
         display: flex;
         gap: 1rem;
         justify-content: center;
-        margin-left: 2rem;
-        margin-top: 1.7rem;
+        padding: 1rem 0;
   
         div {
           border-radius: 50%;
-          border: 1px solid $white;
+          border: $simple-boder-black;
           display: grid;
           height: 3rem;
           margin: 0;
           place-items: center;
           width: 3rem;
           
           a {
-            color: $white;
+            color: $black;
 
             i {
               font-size: 1.2rem;
             } 
           }
         }
+      }
+    }
   
-        div:hover {
-          border: 1px solid $text-grey;
-          cursor: pointer;
+    span {
+      margin-top: 1rem;
+      text-align: center;
+      display: block;
+    }
+  }
+}
 
-          i{
-            color: $text-grey;          
-          }
+@media (min-width: $breakpoint-mobile) {
```

Comment:
Sort all your css properties, in this file

Reply:
Sorted